docs/content/docs/advanced/offline-browsing.md

```diff
@@ -74,7 +74,30 @@ Congratulation. Your website is now a offline-first application.
 
 ## FAQ
 
+### How to enable on demand cache for files that weren't cache on initial load ?
+
+This is the default behavior of the builtin Service Worker logic.
+It will cache everything under the same origin on demand.
+
+### Can you provide other useful globby patterns ?
+
+Of course. Here is some useful pattern that covers 90% cases of Phenomic's user.
+(along with the default pattern)
+
+- `["index.html", "**.css", "**.js"]`: Only cache css, js files and the main `index.html`.
```

Comment:
For situation like #277 , I don't know. Looks like it's the same, but we can make it better since it's our logic.